webapp/channels/src/components/dropdown_input.tsx

```diff
@@ -101,17 +95,37 @@ const DropdownInput = <T extends ValueType>(props: Props<T>) => {
         }
     };
 
+    const {formatMessage} = useIntl();
+    const [customInputLabel, setCustomInputLabel] = useState<CustomMessageInputType>(null);
+    const [ownValue, setOwnValue] = useState<T>();
+
+    const ownOnChange = (value: T, action: ActionMeta<T>) => {
```

Comment:
new function on every render. This happens on many other functions in this file, so we can omit this change for now, or fix it in the whole file. 0/5

Reply:
Noted for followup PR.